src/main/dune

```diff
@@ -1,14 +1,19 @@
 (executables
  (names Main)
+ (flags
+   (:include flags.sexp)
+;   (:standard -open TCB -open Commons_TCB)
+   )
  (libraries
+    TCB
     semgrep_core_cli
     osemgrep_cli
     lwt_platform.unix
  )
  (preprocess (pps ppx_profiling))
  ; for ocamldebug
- (modes native byte)
- (flags (:include flags.sexp))
+ ;XXX(modes native byte)
```

Comment:
@emjin mode natives strike again!
I commented it while experimenting and forgot about it.